packages-exp/functions-exp/src/public-types.ts

```diff
@@ -51,13 +50,12 @@ export interface HttpsCallableOptions {
 }
 
 /**
- * `Functions` represents a Functions instance, and is a required argument for
- * all Functions operations.
+ * An instance of Firebase Functions.
```

Comment:
"Firebase Functions" isn't an actual product name.

Is there a reason not to just go with the literal here, like "A `Functions` instance?"

Reply:
Sounds good, done.